src/main/resources/com/puppycrawl/tools/checkstyle/checks/annotation/messages_de.properties

```diff
@@ -1,17 +1,25 @@
 annotation.incorrect.style=Die Annotation muss dem Stil ''{0}'' folgen.
-annotation.location=Die Annotation ''{0}'' hat eine unerwartete Einrückungstiefe von {1} (erwartet: {2}).
+annotation.location=Die Annotation ''{0}'' hat eine unerwartete Einrückungstiefe von {1} \
+                    (erwartet: {2}).
```

Comment:
Can we control indentation?

Reply:
not sure, I can ask crowdin support about it. Is it a big issue?